### DISCUSSION TITLE
Code syntax incorrect highlighting

### DISCUSSION THREAD

**Discussion by mcpiatkowski** [one-dark/jetbrains-one-dark-theme#277] [General]
Hello,

hope I can raise it here. I have an issue with getting One Dark theme to work. I tried on two different machines (mac and windows) using two different versions of PyCharm (community and professional). In both cases colors don't work as expected.

I tried bunch of solutions found on internet but nothing worked for me.

I attach screenshots of how it should look like and how it actually looks in PyCharm.

![Screenshot 2022-11-30 at 18 16 59](https://user-images.githubusercontent.com/80952559/204876341-d519b2da-9a7f-43c7-9bb8-ae52042bc5aa.png)

![Screenshot 2022-11-30 at 18 10 03](https://user-images.githubusercontent.com/80952559/204876367-7dea7bf1-0ec6-45f3-a685-11eaac01e60a.png)

Has anyone solved it yet? I am desperate! I can't code without this theme!

Thanks a lot!

**Comment by Unthrottled** [comment]
<img width="1151" alt="Screenshot 2022-11-30 at 6 46 49 PM" src="https://user-images.githubusercontent.com/15972415/204939025-005b1e14-5845-462e-b1b3-1f22d972e421.png">

Looks like the tokenization of Python is different in Pycharm as compared to VSCode. 

You are free to play around with `Settings` -> `Editor` -> `Color Scheme` -> `Python` , but you may be limited as to what the Python plugin tokenizes.

**Reply by mcpiatkowski** [reply]
Let me play with it and if I find an interesting solution I will get back here in case someone else will be having this problem.

Thanks a lot!
